docker-image-samples/feedback-to-model/main.py

```diff
@@ -12,7 +12,7 @@ def handler(las_client, event, environ):
     logging.info(f'Sending feedback to the model: {event} {environ}')
     document_id = event['documentId']
     verified = event['verified']
-    ground_truth = [{'label': k, 'value': v} for k, v in verified.items()]
+    ground_truth = [{'label': k, 'value': v} for k, v in verified.items() if v != '']
```

Comment:
If the idea is that empty string throws a deliberate error from the API it seems like a sketchy move to do this filtering here.